### PR TITLE
readme update

### PR DESCRIPTION
**Pull request by Lxkas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Coming soon...
 
 ## 🧑‍💻 Contributors
 - **Tawan Tocharoentanaphol** - [@Lxkas](https://github.com/Lxkas)
+- **Ayham H. Tamim** - [@Ayham1590](https://github.com/Ayham1590)
```
Comment (chinathaip) [inline README.md:53]: why am I not in the contributor list?

Comment (Naris167) [inline README.md:53]: Where is my name LOL?

**Review by chinathaip — APPROVED** [review]
LGTM now that my name is in there despite not having any contributions

File: README.md [hunk 2]
```diff
@@ -59,4 +60,4 @@ This project is licensed under the MIT License - see the [LICENSE](LICENSE) file
 ## 📧 Contact
 For any queries, please feel free to reach out to us at [email](mailto:some-email@example.com).
 
-Make the most of your Stamford experience with us!
\ No newline at end of file
+Make the most of your Stamford experience with us~!
```
Comment (Naris167) [inline README.md]: 🔥🔥🔥

**Review by Naris167 — APPROVED** [review]

**Pull request merged**